packages/cf.js/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "sourceRoot": ".",
     "declarationDir": "dist/",
-    "outDir": "dist/"
+    "outDir": "dist/",
+    "types": ["jest"]
```

Comment:
Why is this needed?

Reply:
So we don't need to import `expect`, `beforeAll`, etc and also so we don't need to rely on a global in typescript-typings

Reply:
We import those? Where?

Reply:
It is kind of subtle at the moment. `@types/jest` happens to get hoisted and in the root tsconfig we have a typeRoots config option that fetches it. However if we explicitly declare this on a per-package level it is safer and does not assume hoisting. Super tiny detail